internal/post/message.go

```diff
@@ -93,18 +105,26 @@ func (msg *Message) Bytes() []byte {
 			fmt.Fprint(sub, part.Content)
 		}
 
-		_ = wr.Close()
+		err := wr.Close()
```

Comment:
This should be deferred after the assignation so guarantee that it always gets close.

Use the same pattern than above `defer func() { err = errs.Combine(err, ...) }()`